code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -78,6 +79,39 @@
 		"is twisting their own neck!",
 		"is holding their breath!")
 
+	var/datum/action/innate/tail_lash/lash = new()
+
+
+/datum/species/unathi/handle_post_spawn(var/mob/living/carbon/human/H)
+
+	lash.Grant(H)
+	..()
+
+/datum/action/innate/tail_lash
+	name = "Tail lash"
+	icon_icon = 'icons/effects/effects.dmi'
+	button_icon_state = "tail"
+
+/datum/action/innate/tail_lash/Activate()
+	var/mob/living/carbon/human/user = owner
+	if(!user.restrained() || !user.buckled)
+		to_chat(user, "You need freedom of movment to tail lash!")
+		return
+	for(var/mob/living/carbon/human/C in orange(1))
+		var/obj/item/organ/external/E = C.get_organ(pick("l_leg", "r_leg", "l_foot", "r_foot", "groin"))
+		if(E)
+			if(user.getStaminaLoss() >= 50)
+				to_chat(user, "Rest before tail lashing again!")
+			else
+				user.changeNext_move(CLICK_CD_MELEE)
+				user.visible_message("<span class='danger'> [src] smacks [C] in the [E] with thier tail!</span>", "<span class='danger'>You hit [C] in the [E] with your tail!</span>")
+				user.adjustStaminaLoss(15)
+				C.apply_damage(5, BRUTE, E)
+				user.spin(20,1)
+				playsound(user.loc, 'sound/weapons/slash.ogg', 50, 0)
```

Comment:
This makes me think this should be a verb on carbons, and given to xenomorphs as well (they have sharp tails, don't they?)

Reply:
Good point